### ISSUE TITLE
UE animation

### ISSUE THREAD

**Issue by chrisgoringe** [chrisgoringe/cg-use-everywhere#137]
The dots aren't animated unless you move the mouse...

**Comment by chrisgoringe** [comment]
Fixed in latest push (plus added pulse mode)

**Comment by JorgeR81** [comment]
They look good !

But in the pulse animation, colors look a bit washed out, at the point of maximum intensity.  

Can you reduce the glow a bit, and give it more color saturation. 

The old glowing wire design had very saturated colors, so the wires were easier to spot ( even without dot animation ).
If you can get a similar look, with this method, it would be nice. 

![anim](https://github.com/chrisgoringe/cg-use-everywhere/assets/72734038/81adcf72-c9a7-42ce-b2cf-309488aae2a9)

**Comment by JorgeR81** [comment]
OK, I see the glow size is controlled by the zoom level, so maybe you can't change that.

But I noticed wires glow white once, then they glow with color.
Maybe wires would look better if they glow with color every time. 
Can you change that ?

![ps1](https://github.com/chrisgoringe/cg-use-everywhere/assets/72734038/1a492602-2b46-45a3-bf10-f9aae0a5ca39)

![ps2](https://github.com/chrisgoringe/cg-use-everywhere/assets/72734038/20090c70-b5d6-405d-8b90-5351d10e8c0c)

**Comment by JorgeR81** [comment]
I think it looks better now !

I also noticed a minor issue.

Some of the pale green wires don't glow, if the UE? node it's minimized.

I need to maximize for this to start working.
And when I minimize again, it still works. 

This was tested with the workflow I shared for the other issue. 

![wires](https://github.com/chrisgoringe/cg-use-everywhere/assets/72734038/bf2c6314-d791-4e72-be8f-063c03176c6f)
